service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java

```diff
@@ -710,11 +735,53 @@ void editNewTariffSuccess() {
         verify(tariffsLocationRepository).saveAll(anySet());
     }
 
+    @Test
+    void addNewTariffThrowsExceptionWhenListOfLocationsIsEmptyTest() {
+        AddNewTariffDto dto = ModelUtils.getAddNewTariffDto();
+
+        when(courierRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getCourier()));
+        when(userRepository.findByUuid("35467585763t4sfgchjfuyetf")).thenReturn(ModelUtils.getUser());
+        when(tariffsInfoRepository.save(any())).thenReturn(ModelUtils.getTariffInfo());
+        when(tariffsLocationRepository.findAllByCourierIdAndLocationIds(1L, List.of(1L)))
+            .thenReturn(Collections.emptyList());
+        when(receivingStationRepository.findAllById(List.of(1L))).thenReturn(ModelUtils.getReceivingList());
+        when(locationRepository.findAllByIdAndRegionId(dto.getLocationIdList(),
+            dto.getRegionId())).thenReturn(Collections.emptyList());
+
+        assertThrows(NotFoundException.class,
+            () -> superAdminService.addNewTariff(dto, "35467585763t4sfgchjfuyetf"));
+
+        verify(courierRepository).findById(1L);
+        verify(userRepository).findByUuid("35467585763t4sfgchjfuyetf");
+        verify(tariffsInfoRepository, times(1)).save(any());
+        verify(receivingStationRepository).findAllById(List.of(1L));
+        verify(locationRepository).findAllByIdAndRegionId(dto.getLocationIdList(), dto.getRegionId());
```

Comment:
please add verify for `tariffsLocationRepository.findAllByCourierIdAndLocationIds(1L, List.of(1L)))`

Reply:
added